src/ui/preview/Playground.tsx

```diff
@@ -168,20 +168,9 @@ add_filter( "determine_current_user", function() { return 1; }, 99999 );
 function deleteDefaultContent(): string {
 	return `<?php
 require_once 'wordpress/wp-load.php';
-$posts = get_posts( array( 'posts_per_page' => -1 ) );
-foreach ( $posts as $post ) {
-    wp_delete_post( $post->ID, true );
-}
-$pages = get_posts(
-    array(
-        'posts_per_page' => -1,
-        'post_type'      => 'page',
-        'post_status'    => array( 'publish', 'draft' ),
-    )
-);
-foreach ( $pages as $page ) {
-    wp_delete_post( $page->ID, true );
-}
+wp_delete_post( 1, true );
+wp_delete_post( 2, true );
+wp_delete_post( 3, true );
```

Comment:
I love this, as simple as it gets and does what we want 👌

Reply:
Only took 2 days of hair pulling on why persistence was broken 😅